src/adera/Machines/UserControl.cpp

```diff
@@ -111,6 +84,11 @@ void SysMachineUserControl::update_sensor(ActiveScene &rScene)
     // InputDevice.IsActivated()
     // Combination
     auto const &usrCtrl = rScene.reg_root_get_or_emplace<ACompUserControl>(rScene.get_user_input());
+    ACompWireNodes<Percent> &rNodesPercent = SysWire::nodes<Percent>(rScene);
+    ACompWireNodes<AttitudeControl> &rNodesAttCtrl = SysWire::nodes<AttitudeControl>(rScene);
+
+    std::vector< nodeindex_t<wiretype::Percent> > updPercent;
+    std::vector< nodeindex_t<wiretype::AttitudeControl> > updAttCtrl;
```

Comment:
Is there any way we could wrap this stuff up into a cleaner interface? `SysWire::get_node<T>(index)` for the node groups? I guess passing the update list into `signal_assign()` isn't terrible, but it would be nice if it were more opaque from the users's point of view.

Reply:
The machines we have so far only have one connection of each type. Future machines may have two percent outputs for example. Having an `SysWire::get_node<T>(index)` would mean obtaining the ACompWireNodes component multiple times in many cases. This will only save one line of code anyways.

For the vector, a `using` may make this cleaner
```cpp
template <typename WIRETYPE_T>
using NodeUpdates_t = std::vector< nodeindex_t<WIRETYPE_T> >;
```